poker-pals/src/App.vue

```diff
@@ -21,15 +22,18 @@ export default {
     // Tables
   }
 }
+
 </script>
 
 <style>
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
+
 #app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
+  font-family: Roboto;
```

Comment:
We should keep at least one common font as backup in case the browser can't render Roboto
```suggestion
  font-family: Roboto, sans-serif;
```